src/stores/workflows.ts

```diff
@@ -48,6 +48,7 @@ const useWorkflowsStore = defineStore('workflows', {
       type: WorkflowType,
       filter: PartialRunTaskFilter | PartialProcessDataFilter,
       body?: string,
```

Comment:
I'm not a big fan of having to pass `undefined` arguments to use a single optional argument later in the list. Maybe pass an options object with optional `body` and `fileName` (or `fileNameSuffix`..) fields?

Reply:
I agree, I wanted named params from Python at this moment

Reply:
Added to an options object